docs/UserGuide.md

```diff
@@ -354,7 +381,8 @@ Example:
 Gets the common timings of persons who are free with the same tag.
 
 Format: `findCommonTiming t/TAG`
-* Show the overlapping timings that a group of friends with the same tags are free
+
+* Show the overlapping timings that a group of friends with the same tags are free.
```

Comment:
I think this part isnt very clear, would "List the time intervals for which contact with the given tag is free" be better?